### PR TITLE
Build: Only run browser tests in one Node version on Travis

### PR DESCRIPTION
**Pull request by mgol**
Ref gh-3744

### Summary ###

Only run browser tests in one Node version on Travis. This reduces the possibility of failure.

### Checklist ###

* [x] All authors have signed the CLA at https://cla.js.foundation/jquery/jquery
* ~~New tests have been added to show the fix or feature works~~
* [x] Grunt build and unit tests pass locally with these changes
* ~~If needed, a docs issue/PR was created at https://github.com/jquery/api.jquery.com~~

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,3 +7,12 @@ node_js:
...
   chrome: stable
+env:
+  - NPM_SCRIPT=test:browserless
+matrix:
+  include:
+    # Run browser tests only on one Node.js version to save time.
+    - node_js: "8"
+      env: NPM_SCRIPT=test
+script:
+  - npm run-script $NPM_SCRIPT
```
Comment (Krinkle) [inline .travis.yml]: Might want to use `npm run` here for consistency with the other callers.

Comment (mgol) [reply .travis.yml]: Done.

**Comment by mgol** [comment]
@Krinkle I modified it as some tests were running twice. Now all Nodes run browserless tests and - *in addition* to that - one Node runs browser tests *only*. PTAL.

**Comment by Krinkle** [comment]
@mgol Yep, makes sense. Also cuts down total time for Travis CI from 13min to 9min.

**Pull request merged**